src/core/reactor.cc

```diff
@@ -2683,9 +2690,13 @@ void reactor::register_metrics() {
             // total_operations value:DERIVE:0:U
             sm::make_counter("fsyncs", _fsyncs, sm::description("Total number of fsync operations")),
             // total_operations value:DERIVE:0:U
-            sm::make_counter("io_threaded_fallbacks", std::bind(&thread_pool::operation_count, _thread_pool.get()),
-                    sm::description("Total number of io-threaded-fallbacks operations")),
-
+            io_fallback_counter("aio_fallback", submit_reason::aio_fallback),
+            // total_operations value:DERIVE:0:U
```

Comment:
do you even understand these comments? I sure don't.

Reply:
Haha, it's collectd stuff: https://www.collectd.org/documentation/manpages/types.db.html#:~:text=db%20file%20contains%20collectd's%20metric,bytes%E2%80%9D%20and%20%E2%80%9Ctotal_bytes%E2%80%9D.

I was wondering the same in the past